src/Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs

```diff
@@ -568,19 +560,15 @@ private string GetUsingTemplate(Language language)
             {
                 case Language.CSharp:
                     return
-                        "using System;\n" +
```

Comment:
This is breaking change compare to ps 5.1

Reply:
I'd revert this but the _whole_ cmdlet is a breaking change. So it makes no sense to revert.

Reply:
Fixed.